### DISCUSSION TITLE
Speed limits

### DISCUSSION THREAD

**Discussion by Hanbao-Deguo** [Stephan-S/FS19_AutoDrive#2056] [Ideas]
I often have to pay money, because my AD driver is too fast and gets a ticket from a speed trap. It would great to be able to define a *road* speed limit just to avoid such costs. I would also use them for my field trails.

**Comment by Axel32019** [comment]
@Hanbao-Deguo Hier schaut nicht wirklich jemand rein, daher bitte Feature-Request Issue erstellen.

**Comment by crcarmichael** [comment]
@Hanbao-Deguo speed limits are already a feature. You can set the field limit and the road limit in the HUD. What map has speed traps?!

**Comment by Hanbao-Deguo** [comment]
Altkirch im Elsass has speed trap.

I also would like to limit the speed in cities, when a sign limits the speed, and on the ways beside fields.

Mit freundlichen Grüßen

Christoph Schreiber
Pettenkoferstraße 34
34121 Kassel
0179 / 328 7617

Am 30.09.2021 um 16:33 schrieb crcarmichael ***@***.******@***.***>>:



@Hanbao-Deguo<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FHanbao-Deguo&data=04%7C01%7C%7C4393b85006924bee13f908d9841f4daf%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637686092254307266%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=cK0pP%2Bf3Boj%2B7OTtHFugXEsbVjtd7HdpQCeIzyA%2BEg0%3D&reserved=0> speed limits are already a feature. You can set the field limit and the road limit in the HUD. What map has speed traps?!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FStephan-S%2FFS19_AutoDrive%2Fdiscussions%2F2056%23discussioncomment-1409429&data=04%7C01%7C%7C4393b85006924bee13f908d9841f4daf%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637686092254307266%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=re2kKj%2FEImxsrufkmPUffwHHp4AZPmwjg3M57OLMXd0%3D&reserved=0>, or unsubscribe<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAGMPEYUUE2VC3UNI2TTHYU3UERYMNANCNFSM5CMBO5TA&data=04%7C01%7C%7C4393b85006924bee13f908d9841f4daf%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637686092254317260%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=tqjRk2YLVb5uKTDqBEtJDFgWZzMGsccvQ5jRjy7nquk%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7C4393b85006924bee13f908d9841f4daf%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637686092254317260%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=tjxKZ2DNOAYI9Hv5fkcHAt7ohPhqonYCPiYtxGOhi9I%3D&reserved=0> or Android<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7C4393b85006924bee13f908d9841f4daf%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637686092254327255%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=kYDQ4GBW5f759UUVBjylKT9KXkf6kJl%2BUy%2FsbGeI0ZI%3D&reserved=0>.
